front/app/containers/CustomPageShow/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+// components
+import CustomPageHeader from './CustomPageHeader';
+import TopInfoSection from './TopInfoSection';
+import { Container, Content } from 'containers/LandingPage';
+
+// hooks
+import usePage from 'hooks/usePage';
+import { useParams } from 'react-router-dom';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
+import { ICustomPagesAttributes } from 'services/customPages';
+
+const LandingPage = () => {
+  const { slug } = useParams() as {
+    slug: string;
+  };
+  const customPage = usePage({ pageSlug: slug });
+
+  if (isNilOrError(customPage)) {
+    return null;
+  }
+
+  const attributes = customPage.attributes as unknown as ICustomPagesAttributes;
```

Comment:
We have a useCustomPage hook for custom pages. You will need to extend it with the option to pass a slug.